### DISCUSSION TITLE
Changing output of summary statistics into a dataframe

### DISCUSSION THREAD

**Discussion by wangarikiarie** [CamDavidsonPilon/lifelines#1354] [Q&A] [answered]
Hello!

I was wondering if anyone had some thoughts on this. I have fitted a CoxPHFitter to my data and have the summary results. I want to change the summary statistics into a data frame so that I can pick the variables which are significant. Is there a way to do this in lifelines?

The equivalent of this in a regression would be changing it to HTML and then reading it. Something along the lines of:

```
results_summary = results.summary()
results_as_html = results_summary.tables[1].as_html()
pd.read_html(results_as_html, header=0, index_col=0)[0]
```

**Comment by CamDavidsonPilon** [comment, marked as answer]
Hi @wangarikiarie, 

does `results.summary` give you what you are looking for?  https://github.com/CamDavidsonPilon/lifelines/blob/master/lifelines/fitters/coxph_fitter.py#L2167

**Reply by wangarikiarie** [reply]
Absolutely! This works. Thank you!
